website/docs/sdk-reference/dart.md

```diff
@@ -130,34 +158,68 @@ final user = ConfigCatUser(
 );
 ```
 
-## Polling Modes
-The *ConfigCat SDK* supports 3 different polling mechanisms to acquire the setting values from *ConfigCat*. After latest setting values are downloaded, they are stored in the internal cache then all `getValue()` calls are served from there. With the following polling modes, you can customize the SDK to best fit to your application's lifecycle.  
-[More about polling modes.](/advanced/caching/)
+### Default user
 
-### Auto polling (default)
-The *ConfigCat SDK* downloads the latest values and stores them automatically every 60 seconds.
+There's an option to set a default user object that will be used at feature flag and setting evaluation. It can be useful when your application has usually a single user only, or rarely switches users.
+
+You can set the default user object either on SDK initialization:
 
-Use the the `autoPollInterval` option parameter of the `PollingMode.autoPoll()` to change the polling interval.
 ```dart
 final client = ConfigCatClient.get(
-    sdkKey: '<PLACE-YOUR-SDK-KEY-HERE>',
+    sdkKey: '#YOUR-SDK-KEY#',
     options: ConfigCatOptions(
-        mode: PollingMode.autoPoll(
-            autoPollInterval: Duration(seconds: 100),
-        ),
+        defaultUser: ConfigCatUser(identifier: 'john@example.com')
     )
 );
 ```
-Adding a callback to `onConfigChanged` option parameter will get you notified about changes.
+or with the `setDefaultUser()` method of the ConfigCat client.
+```dart
+client.setDefaultUser(ConfigCatUser(identifier: 'john@example.com'));
+```
+
+If the default user is set, the SDK will use it every time when the `getValue()`, `getValueDetails()`, `getAllValues()`, or `getAllVariationIds()` methods are called without passing a user object to them.
```

Comment:
A suggestion:

Whenever the `getValue()`, `getValueDetails()`, `getAllValues()`, or `getAllVariationIds()` methods are called without an explicit user object parameter, the SDK will automatically use the default user as a user object.

Reply:
✅